include/fiction/technology/sidb_charge_state.hpp

```diff
@@ -122,6 +123,124 @@ charge_configuration_to_string(const std::vector<sidb_charge_state>& charge_dist
 
     return config_str.str();
 }
+/**
+ * The iteration direction over the SiDB states. , and in reverse to the
+ * conductance band.
+ */
+enum class sidb_state_iter_dir
+{
+    /**
+     * To the valance band iterates "\f$-\to 0\to +\f$".
+     */
+    TO_VALENCE_BAND,
```

Comment:
I don't know if I like the naming here. Can we choose different ones without the names ``valence`` and ``conductance``.

Reply:
`TO_POSITIVE` and `TO_NEGATIVE`?

Reply:
I prefer it!

Reply:
Implemented